megatron/model/fused_rmsnorm.py

```diff
@@ -1,13 +1,20 @@
+from megatron import get_args
+
 import torch
 from torch.nn.parameter import Parameter
+from torch.nn import init
 import intel_extension_for_pytorch as ipex  # noqa
 
 # Taken from facebookresearch/llama
 class RMSNorm(torch.nn.Module):
-    def __init__(self, dim: int, eps: float = 1e-6):
+    def __init__(self, dim: int, eps: float = 1e-6, sequence_parallel=False):
         super().__init__()
         self.eps = eps
-        self.weight = Parameter(torch.ones(dim))
+        self.weight = Parameter(torch.empty(dim,
```

Comment:
Why split weight initialization into two steps? What is wrong with the following?

```suggestion
        self.weight = Parameter(torch.ones(dim,
```

Reply:
Modified, yes, is no need to init twice, torch.ones is okay.